pkg/manifestgen/install/install.go

```diff
@@ -35,17 +35,13 @@ import (
 // Generate returns the install manifests as a multi-doc YAML.
 // The manifests are built from a GitHub release or from a
 // Kustomize overlay if the supplied Options.BaseURL is a local path.
-func Generate(options Options) (*manifestgen.Manifest, error) {
+func Generate(options Options, manifestsBase string) (*manifestgen.Manifest, error) {
```

Comment:
Any specific reason everything is an option except for the base?

Reply:
Because the base can be created by the CLI only, since you can't embed files in a module and TF can't deal with it.

Reply:
Let me put this way, the install.Options are meant to be mapped in TF/eksctl/etc as user-supplied inputs, the `manifestsBase` doesn't fit into this category.